examples/example_FastMRI_Unet.py

```diff
@@ -0,0 +1,206 @@
+    axs[0].set_title("MR Image")
+    axs[1].scatter(*traj.T, s=0.5)
+    axs[1].set_title("Trajectory")
+    axs[2].imshow(np.abs(recon[0][0].detach().cpu().numpy()), cmap="gray")
+    axs[2].axis("off")
+    axs[2].set_title("Reconstruction")
+    if loss is not None:
+        axs[3].plot(loss)
+        axs[3].grid("on")
+        axs[3].set_title("Loss")
+    if save_name is not None:
+        plt.savefig(save_name, bbox_inches="tight")
+        plt.close()
+    else:
+        plt.show()
+
+
+# %%
+# Setup Inputs (models, trajectory and image)
+init_traj = initialize_2D_spiral(64, 256).reshape(-1, 2).astype(np.float32)
+model = Model(init_traj)
+model.eval()
+
+# %%
+# The image on which we are going to train.
+mri_2D = torch.Tensor(np.flipud(bwdl.get_mri(4, "T1")[80, ...]).astype(np.complex64))[
+    None
+]
+mri_2D = mri_2D / torch.mean(mri_2D)
+kspace_mri_2D = model.operator.op(mri_2D)
+
+# Before training, here is the simple reconstruction we have using a
+# density compensated adjoint.
+old_recon = model(kspace_mri_2D)
+fig, axs = plt.subplots(1, 3, figsize=(15, 5))
+plot_state(axs, mri_2D, init_traj, old_recon)
+
+
+# %%
+# Start training loop
+epoch = 100
+optimizer = torch.optim.RMSprop(model.parameters(), lr=1e-3)
+losses = []  # Store the loss values and create an animation
+image_files = []  # Store the images to create a gif
+model.train()
+
+with tqdm(range(epoch), unit="steps") as tqdms:
+    for i in tqdms:
+        out = model(kspace_mri_2D)  # Forward pass
+
+        loss = torch.nn.functional.l1_loss(out, mri_2D[None])  # Compute loss
+        tqdms.set_postfix({"loss": loss.item()})  # Update progress bar
+        losses.append(loss.item())  # Store loss value
+
+        optimizer.zero_grad()  # Zero gradients
+        loss.backward()  # Backward pass
+        optimizer.step()  # Update weights
+
+        # Generate images for gif
+        hashed = joblib.hash((i, "learn_traj", time.time()))
+        filename = "/tmp/" + f"{hashed}.png"
+        fig, axs = plt.subplots(2, 2, figsize=(10, 10))
+        plot_state(
+            axs,
+            mri_2D,
+            init_traj,
+            out,
+            losses,
+            save_name=filename,
+        )
+        image_files.append(filename)
+
+
+# Make a GIF of all images.
+imgs = [Image.open(img) for img in image_files]
+imgs[0].save(
+    "mrinufft_learn_unet.gif",
+    save_all=True,
+    append_images=imgs[1:],
+    optimize=False,
+    duration=2,
+    loop=0,
+)
+
+# Cleanup
+for f in image_files:
+    try:
+        os.remove(f)
+    except OSError:
+        continue
+# don't raise errors from pytest.
+# This will only be executed for the sphinx gallery stuff
+
+try:
+    final_dir = (
+        Path(os.getcwd()).parent.parent
+        / "docs"
+        / "generated"
+        / "autoexamples"
+        / "GPU"
+        / "images"
+    )
+    shutil.copyfile("mrinufft_learn_unet.gif", final_dir / "mrinufft_learn_unet.gif")
+except FileNotFoundError:
+    pass
```

Comment:
Please surround this with sphinx ignore tags, it will prevent it from appearing in docs.

See here: https://github.com/mind-inria/mri-nufft/blob/092c795c9fc251966ce0422ee40eb4419ff4faf1/examples/GPU/example_learn_samples.py#L165-L190 

The corresponding example: https://mind-inria.github.io/mri-nufft/generated/autoexamples/GPU/example_learn_samples.html#sphx-glr-generated-autoexamples-gpu-example-learn-samples-py